app.py

```diff
@@ -78,7 +78,10 @@ def get_relative_path(page_name):
     sidebar,
     content,
     dummy_div,
-    dcc.Store(id='stored-data')
+    dcc.Store(id='stored-data'),
+    dcc.Store(id='stored-data-func'),
+    dcc.Store(id='stored-data-education-total'),
+    dcc.Store(id='stored-data-education-outcome')
```

Comment:
Since these two are only referenced in `education.py` consider moving the declarations there?

Reply:
Yes.. I have been going back and forth about where to put these lines.. 
If I put it in app.py, I can share the cache across pages (no need to query education if it is the second time rendering). 
But this also causes unnecessary queries on the home screen. 

There does not seem to be visible differences right now. let's move it back to education.py :)